pkg/pool/operations/create.go

```diff
@@ -59,7 +59,6 @@ func (oc *OperationsConfig) Create(cspi *cstor.CStorPoolInstance) error {
 		}
 		// Remove this raidGroup
 		cspiCopy.Spec.DataRaidGroups = append(cspiCopy.Spec.DataRaidGroups[:i], cspiCopy.Spec.DataRaidGroups[i+1:]...)
-		break
```

Comment:
This `break` is required
 
**Context**: When the user specifies multiple `DataRaidGroups` first we will create a pool(at line 55 using 0th index raid group) and we will remove used RaidGroup from the total group list and then expand the pool.